stubs/tests/ResourceTest.php

```diff
@@ -29,7 +29,9 @@ public function test_users_can_create_resources()
             'summary' => 'This is my resource.',
         ]);
 
-        $url = localized_route('resources.show', ['resource' => Str::slug($user->id)]);
+        $resource = Resource::where('title', 'Test resource')->first();
+
+        $url = localized_route('resources.show', ['resource' => $resource]);
```

Comment:
I'd suggest using the shorthand here:

```suggestion
        $url = localized_route('resources.show', $resource);
```